### PR TITLE
ubuntu 22.04: update kernel to SPR-BKC-PC-v9.6

### PR DESCRIPTION
**Pull request by jialeif**
And clear unused patch.

Signed-off-by: jialeie <jialei.feng@intel.com>

### CODE REVIEW

File: build/ubuntu-22.04/intel-mvp-spr-kernel/build.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ CURR_DIR=$(dirname "$(readlink -f "$0")")
 UPSTREAM_VERSION="5.15"
 DOWNSTREAM_GIT_URI="https://github.com/intel/linux-kernel-dcp.git"
 
-DOWNSTREAM_TAG="SPR-BKC-PC-v8.8"
+DOWNSTREAM_TAG="SPR-BKC-PC-v9.6"
```
Comment (clsulliv) [inline build/ubuntu-22.04/intel-mvp-spr-kernel/build.sh:9]: It looks like this tag has not yet been pushed to the linux-kernel-dcp repo?

Comment (ruomengh) [reply build/ubuntu-22.04/intel-mvp-spr-kernel/build.sh:9]: The tag has been pushed to public repo

**Comment by jialeif** [comment]
Verified this PR on Ubuntu 22.04

**Review by ruomengh — APPROVED** [review]
LGTM

**Pull request merged**